tasks.py

```diff
@@ -234,43 +234,44 @@ def load_addresses(ctx, files=[]):
         logging.info("no addresses to import")
         return
 
+    output_csv = "output.csv"
+
+    options = []
     if addr_config.get("bano", {}).get("file"):
-        logging.info("importing bano addresses")
-        conf = ctx.addresses.bano
-        additional_params = _get_cli_param(conf.get("nb_threads"), "--nb-threads")
-        additional_params += _get_cli_param(conf.get("nb_shards"), "--nb-shards")
-        additional_params += _get_cli_param(conf.get("nb_replicas"), "--nb-replicas")
-        run_rust_binary(
-            ctx,
-            "mimir",
-            "bano2mimir",
-            files,
-            "--input {ctx.addresses.bano.file} \
-            --connection-string {ctx.es} \
-            {additional_params} \
-            --dataset {ctx.dataset}".format(
-                ctx=ctx, additional_params=additional_params
-            ),
-        )
```

Comment:
we should keep the possibility to use `bano2mimir` for backwards compatibility

Reply:
Why would we need backwards compatibility?

Reply:
To be able to reuse existing pipeline configuration with the new tasks definitions